File: new_exercises_11_11_2025/first_non_repititive_character.py
def get_non_repitition(input):
	unique_letter = ""
	checker = ""
	count = 0
	for char in input:
		checker = char
		for letter in input:
			unique_letter = letter
			if char == unique_letter:
				count+=1
		if count == 1:
			return char
			break
		count = 0

word = "dmadam"
print(get_non_repitition(word))

File: new_exercises_11_11_2025/FirstNonRepititiveCharacter.java
import java.util.Scanner;

public class FirstNonRepititiveCharacter{

	public static void getLetter(String userInput){
		int counting = 0;
		for (int count = 0; count < userInput.length(); count++){
			char letterOne = userInput.charAt(count);
			char checker = letterOne;
			for (int counter = 0; counter < userInput.length(); counter++){
				char letterTwo = userInput.charAt(counter);
				if (letterOne == letterTwo){
					counting++;
				}
			}
			if (counting == 1){
				System.out.print(letterOne);
				break;
			}
			counting = 0;
		}
	}
	
	public static void main(String[] args){
		Scanner inputCollector = new Scanner(System.in);

		System.out.print("Enter a word: ");
		String userInput = inputCollector.nextLine();
		

		getLetter(userInput);
		
	}
}